demos/noise_suppression_demo/python/noise_suppression_demo.py

```diff
@@ -134,16 +135,11 @@ def main():
         infer_request_ptr.infer()
         res = infer_request_ptr.output_blobs
 
-        t1 = time.perf_counter()
-
-        samples_times.append(t1-t0)
+        total_latency += perf_counter() - start_time
         samples_out.append(res["output"].buffer.squeeze(0))
 
-    log.info("Sequence of length {:0.2f}s is processed by {:0.2f}s".format(
-        sum(s.shape[0] for s in samples_out)/16000,
-        sum(samples_times),
-
-    ))
+    log.info("Metrics report:")
+    log.info("\tLatency: {:.1f} ms".format(total_latency * 1e3))
```

Comment:
@Wovchena @vladimir-dudnik Here we got ambiguous metric:
Here and before latency is calculated as a total inference time. But to process an audio demo runs model several time, it depends on model input size and audio length. So what is more interesting - time of single inference or  whole time of audio processing?

Reply:
in openvino [FAQ](https://docs.openvinotoolkit.org/latest/openvino_docs_performance_benchmarks_faq.html) latency is defined for synchronous execution mode, and in DL WorkBench [docs](https://docs.openvinotoolkit.org/latest/workbench_docs_Workbench_DG_Key_Concepts.html) it is additionally notes, that latency is measured for single unit of work. Throughput metric, in contrast, measured as number of units of work processed per defined period of time (for example, frames per second).

Reply:
My perception was that we agreed on what we measure two weeks ago and this PR follows this. Moreover README clearly defines: `**Latency**: total processing time required to process input data (from reading the data to displaying the results).`, so there is no ambiguity.

Reply:
ok, then